wranglertools/import_data.py

```diff
@@ -1527,14 +1550,29 @@ def _verify_and_return_item(item, connection):
     return res
 
 
-def cabin_cross_check(connection, patchall, update, infile, remote, lab=None, award=None):
-    """Set of check for connection, file, dryrun, and prompt."""
+def check_and_return_input_type(inputname):
+    if pp.Path(inputname).is_file():
+        xlsx_mime = 'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet'
```

Comment:
This is probably a good candidate for a constant, as we've seen issues with this mime in the past.